### ISSUE TITLE
about the rotation logits q

### ISSUE THREAD

**Issue by dannilv-lab** [brjathu/SKD#2]
I notice that the output of the model only contains the feature and prediction logits (p). how can the rotation logits(q) be obtained?
